definitions/json/ComplexTypes/ComplexTypes-DCF2-nonprod.json

```diff
@@ -0,0 +1,317 @@
+  },
+  {
+    "ID": "DocumentImage",
+    "ListElementCode": "coordinateX",
+    "FieldType": "Number",
+    "ElementLabel": "X position of the image",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "DocumentImage",
+    "ListElementCode": "coordinateY",
+    "FieldType": "Number",
+    "ElementLabel": "Y position of the image",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Bundle",
+    "ListElementCode": "id",
+    "FieldType": "Text",
+    "ElementLabel": "Bundle ID",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Bundle",
+    "ListElementCode": "title",
+    "FieldType": "Text",
+    "ElementLabel": "Name of bundle",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Bundle",
+    "ListElementCode": "description",
+    "FieldType": "TextArea",
+    "ElementLabel": "Description",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Bundle",
+    "ListElementCode": "eligibleForStitching",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Is this the bundle you want to stitch?",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Bundle",
+    "ListElementCode": "eligibleForCloning",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Is this the bundle you want to clone?",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Bundle",
+    "ListElementCode": "documents",
+    "FieldType": "Collection",
+    "ElementLabel": "Bundle document",
+    "FieldTypeParameter": "BundleDocument",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Bundle",
+    "ListElementCode": "folders",
+    "FieldType": "Collection",
+    "ElementLabel": "Bundle folder",
+    "FieldTypeParameter": "BundleFolder",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Bundle",
+    "ListElementCode": "stitchStatus",
+    "FieldType": "Text",
+    "ElementLabel": "Stitch Status",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Bundle",
+    "ListElementCode": "stitchedDocument",
+    "FieldType": "Document",
+    "ElementLabel": "Stitched Document",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Bundle",
+    "ListElementCode": "hasCoversheets",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Should this bundle have coversheets separating each document?",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Bundle",
+    "ListElementCode": "hasTableOfContents",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Should this bundle have a title page with a table of contents?",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Bundle",
+    "ListElementCode": "hasFolderCoversheets",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Should this bundle’s folders have a coversheet?",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Bundle",
+    "ListElementCode": "stitchingFailureMessage",
+    "FieldType": "Text",
+    "ElementLabel": "Error from Stiching service",
```

Comment:
I imagine a callback is going to overwrite this anyway but spelling mistake in stitching